packages/i/ispc/package.yml

```diff
@@ -1,50 +1,37 @@
 name       : ispc
-version    : 1.19.0
+version    : 1.22.0
 release    : 10
 source     :
-    - git|https://github.com/ispc/ispc.git : v1.19.0
+    - git|https://github.com/ispc/ispc.git : v1.22.0
 homepage   : https://github.com/ispc/ispc
 license    : BSD-3-Clause
 component  : programming
 summary    : Intel SPMD Program Compiler
 description: |
     ispc is a compiler for a variant of the C programming language, with extensions for single program, multiple data programming. Under the SPMD model, the programmer writes a program that generally appears to be a regular serial program, though the execution model is actually that a number of program instances execute in parallel on the hardware.
+clang      : yes
 builddeps  :
+    - pkgconfig(LLVMSPIRVLib)
     - pkgconfig(libze_loader)
     - pkgconfig(tbb)
     - vc-intrinsics
-    - spirv-llvm-translator-15-devel
-    - llvm-clang-15-devel
-rundeps    :
-    # For libomp.so
-    # The difference between LLVM 15 and 16 libomp.so is not large enough to
-    # prevent us from using it, and the symbols we use have basically been there
-    # forever.
-    - llvm
-environment: |
-    export CMAKE_PREFIX_PATH=$(llvm-config-15 --prefix)
-    export LDFLAGS="$LDFLAGS -lzstd -lz"
-
-    if [[ " ${PATH[*]} " =~ "ccache" ]]; then
-        export CMAKE_C_COMPILER_LAUNCHER=ccache
-        export CMAKE_CXX_COMPILER_LAUNCHER=ccache
-    fi
+checkdeps  :
+    - intel-compute-runtime-devel
 setup      : |
-    %patch -p1 -i $pkgfiles/0001-Fix-build-with-GCC-13-and-possibly-Clang-15
-
     %cmake_ninja -L \
       -DARM_ENABLED=OFF \
       -DISPC_INCLUDE_EXAMPLES=OFF \
-      -DISPC_INCLUDE_TESTS=OFF \
-      -DISPC_INCLUDE_XE_EXAMPLES=OFF \
-      -DISPC_NO_DUMPS=ON \
+      -DISPC_INCLUDE_TESTS=ON \
       -DISPC_OPAQUE_PTR_MODE=OFF \
-      -DISPC_PREPARE_PACKAGE=OFF \
-      -DISPCRT_BUILD_TESTS=OFF \
-      -DXE_ENABLED=ON
+      -DXE_ENABLED=ON \
+      -DXE_DEPS_DIR=/usr
 build      : |
     %ninja_build
 install    : |
     %ninja_install
-    # ???
-    rm -fr $installdir/home
+    # ??
```

Comment:
Not sure if `??` really has the same emotional impact as `???`.

Reply:
There you go `?!?` :)

Reply:
This is the kind of quality packaging I live for